assets/scss/style-gutenburg.scss

```diff
@@ -1,368 +1 @@
-        top     : 170px;
-        padding : 0 7px;
-      }
-    }
-  }
-
-  /* end LearnDash mitigation routine */
-
-  /*
-  Grouped Items blocks admin styles
-  */
-
-  .block-editor-block-list__layout {
-
-    [data-type="nhsblocks/rowgroup"] .nhsuk-grid-row {
-
-      border : 1px dashed $color_bright-blue;
-
-      &:before {
-        content          : 'Grouped Items';
-        background-color : $color_bright-blue;
-        color            : $color_snow-white;
-        position         : absolute;
-        top              : 0;
-        left             : 0;
-        font-size        : 12px;
-        padding          : 2px 5px;
-      }
-
-      .block-editor-block-list__layout {
-
-        & > div {
-          clear : both;
-        }
-
-        [data-type="nhsblocks/onethird"],
-        [data-type="nhsblocks/onehalf"],
-        [data-type="nhsblocks/twothirds"],
-        [data-type="nhsblocks/onequarter"],
-        [data-type="nhsblocks/threequarters"] {
-          padding : 0;
-          border  : 1px dashed $color_black;
-          clear   : none;
-
-          &.has-child-selected {
-            border   : 3px solid $color_warm-yellow;
-            position : relative;
-            z-index  : 5;
-          }
-        }
-
-      }
-
-      %innerBlocks {
-        float : none;
-        width : 100%;
-      }
-
-      [data-type="nhsblocks/onethird"] {
-        @extend .nhsuk-grid-column-one-third;
-
-        .nhsuk-grid-column-one-third {
-          @extend %innerBlocks;
-        }
-
-      }
-
-      [data-type="nhsblocks/onehalf"] {
-        @extend .nhsuk-grid-column-one-half;
-
-        .nhsuk-grid-column-one-half {
-          @extend %innerBlocks;
-        }
-
-      }
-
-      [data-type="nhsblocks/twothirds"] {
-        @extend .nhsuk-grid-column-two-thirds;
-
-        .nhsuk-grid-column-two-thirds {
-          @extend %innerBlocks;
-        }
-
-      }
-
-      [data-type="nhsblocks/onequarter"] {
-        @extend .nhsuk-grid-column-one-quarter;
-
-        .nhsuk-grid-column-one-quarter {
-          @extend %innerBlocks;
-        }
-
-      }
-
-      [data-type="nhsblocks/threequarters"] {
-        @extend .nhsuk-grid-column-three-quarters;
-
-        .nhsuk-grid-column-three-quarters {
-          @extend %innerBlocks;
-        }
-      }
-    }
-
-  }
-
-  /* make the table cells more obvious when editing */
-  .is-selected {
-    figure.is-style-nhsuk-table-responsive {
-      thead, tr, th, td {
-        border    : 1px dashed $color_grey-3;
-        min-width : 2rem;
-      }
-    }
-  }
-
-
-  /* clearfix after grouped items to sort out some display wirdiness */
-  .block-editor-inner-blocks {
-    @include clearfix();
-  }
-
-  // Download Monitor plugin styling to match NHSUK design patterns
-  .dlmPreviewButton {
-    padding: 0;
-    min-height: 50px;
-    border: 2px dashed $color_grey-1;
-    iframe {
-      min-height: 50px;
-    }
-  }
-  @import 'color-mixer';//gutenberg page style overrides
-
-}
-
```

Comment:
Just wanted to check as we may still need this file. The gutenberg style file is what styles some elements in the backend page editor. For example the @import 'typography-gutenberg'; brings in PT Sans font. Just thinking we may want to keep some of the styles.

Reply:
I moved the font import to the gds-design-system-settings file.  Removing this didn't seem to adversely affect anything.  

gds-design-system-settings.scss starts thus:
```
$govuk-font-family: 'PT Sans', sans-serif;
@import "node_modules/govuk-frontend/govuk/all";
@import url(https://fonts.googleapis.com/css2?family=PT+Sans:ital,wght@0,400;0,700;1,400;1,700&display=swap);